source/class/qx/tool/cli/Cli.js

```diff
@@ -110,14 +111,11 @@ qx.Class.define("qx.tool.cli.Cli", {
       var title = "qooxdoo command line interface";
       title = "\n" + title + "\n" + "=".repeat(title.length);
 
-      // NOTE:: We CANNOT get the framework version here because we will not know which framework
-      //  to load until we have parse the command line args
-
-      title +=
-`
-Versions: @qooxdoo/compiler    v${qx.tool.compiler.Version.VERSION}
+      title += `
+Version: v${await qx.tool.config.Utils.getQxVersion()}
 `;
       title += "\n";
+      // noinspection HtmlDeprecatedTag
```

Comment:
Could you explain this comment?

Reply:
That's my IntelliJ IDE complaining about the use of `<commands>` in the help text, which it thinks is a HTML tag - this is to switch the warning off. But I can remove it.

Reply:
Done.